i18n.php

```diff
@@ -921,7 +921,7 @@
     'but' => 'Nhưng',
     'examples' => 'Dữ liệu',
     'feature' => 'Tính năng',
-    'given' => 'Biết|Cho',
+    'given' => '*|Biết|Cho',
```

Comment:
update_i18n 
https://raw.github.com/cucumber/gherkin/master/lib/gherkin/i18n.json 
404 not found.